process/message.go

```diff
@@ -85,6 +86,31 @@ const (
 	TypeCollectorService           = 44
 )
 
+func (m MessageType) String() string {
+	switch m {
+	case TypeCollectorProc:
+		return "process"
+	case TypeCollectorConnections:
+		return "network"
+	case TypeCollectorRealTime:
+		return "process-rt"
+	case TypeCollectorContainer:
+		return "container"
+	case TypeCollectorContainerRealTime:
+		return "container-rt"
+	case TypeCollectorPod:
+		return "pod"
+	case TypeCollectorReplicaSet:
+		return "replica-set"
+	case TypeCollectorDeployment:
+		return "deployment"
+	case TypeCollectorService:
+		return "service"
+	}
+	// otherwise convert the type identifier
+	return strconv.Itoa(int(m))
```

Comment:
nit: to comply more with switch case, why wasn't this in the default case 🤔 ? 
I don't think there is much of a difference so either should be fine.

Reply:
I was execting Go to complain about a misisng `return` but apparently it works wells 👍